README.md

```diff
@@ -30,7 +30,8 @@ If you need additional help consider reaching out with one of the following opti
 
 - Ask a question in the [``#cloud-init`` IRC channel on Libera](https://kiwiirc.com/nextclient/irc.libera.chat/cloud-init)
 - Search the cloud-init [mailing list archive](https://lists.launchpad.net/cloud-init/)
-- Better yet, join the [cloud-init mailing list](https://launchpad.net/~cloud-init) and participate
+* Follow announcements or ask a question on [Discourse](https://discourse.ubuntu.com/c/server/cloud-init/)
```

Comment:
Adding discourse to this PR because I think it's easier to link to our docs in `config.yml` rather than linking to a any where else or trying to add a ton of links to the issues page.

Reply:
Good idea - after our discussions in Prague I think adding it both here and in the docs index is the right move! Although in both cases I'd change the link text from [Discourse] to [the cloud-init Discourse forum] (longer/more contextual links are better from an accessibility standpoint)